src/common/os.cc

```diff
@@ -294,4 +294,15 @@ NotifyFd::tryRead() const {
     return true;
 }
 
+FdScopeGuard::FdScopeGuard(int &fd)
+    : fd_(fd)
+{
+}
+
+FdScopeGuard::~FdScopeGuard()
+{
+    if(fd_ > 2) {
```

Comment:
Simply `>= 0`. Nothing guarantees that fd won't be 1. Nothing forces standard output, error and input to be open, and they are not sometimes.